src/rpc/methods/f3.rs

```diff
@@ -157,24 +161,33 @@ impl GetPowerTable {
         ctx: &Ctx<impl Blockstore + Send + Sync + 'static>,
         ts: &Arc<Tipset>,
     ) -> anyhow::Result<Vec<F3PowerEntry>> {
+        const BLOCKSTORE_CACHE_CAP: usize = 65536;
```

Comment:
Let's note the memory overhead as a comment here.

Reply:
Done.